rosbag2_cpp/package.xml

```diff
@@ -19,6 +22,9 @@
   <depend>rosidl_typesupport_introspection_cpp</depend>
   <depend>shared_queues_vendor</depend>
 
+  <exec_depend>rosidl_runtime_c</exec_depend>
+  <exec_depend>rosidl_runtime_cpp</exec_depend>
```

Comment:
Since the new dependencies are added as both they can use a `depend` tag.

Reply:
`rosidl_runtime_cpp` is not generating any `.so/dll/dylib` should not be a `<depend>`, right? only `<build_depend>` and `<build_export_depend>` ?

Reply:
new changes applied: 

`rosidl_runtime_c` moved to `<depend>`.

Headers are used here https://github.com/ros2/rosbag2/blob/ec4fb3913233df00850b53a4422d9b804fbab9d7/rosbag2_cpp/include/rosbag2_cpp/converter_interfaces/serialization_format_serializer.hpp#L24

-----

`rosidl_runtime_cpp` moved to `<build_depend>` and `<build_export_depend>`

https://github.com/ros2/rosbag2/blob/ec4fb3913233df00850b53a4422d9b804fbab9d7/rosbag2_cpp/include/rosbag2_cpp/typesupport_helpers.hpp#L24